diff -urN nagios-plugin-check_raid-4.0.8.org/check_raid.cfg nagios-plugin-check_raid-4.0.8/check_raid.cfg
--- nagios-plugin-check_raid-4.0.8.org/check_raid.cfg	2017-09-01 20:58:44.000000000 +0200
+++ nagios-plugin-check_raid-4.0.8/check_raid.cfg	2018-12-11 15:37:33.361179565 +0100
@@ -2,7 +2,7 @@
 # check_raid
 define command {
 	command_name    check_raid
-	command_line    /usr/lib/nagios/plugins/check_raid $ARG1$
+	command_line    /usr/lib/nagios/plugins/check_raid --bbu-monitoring $ARG1$
 }
 
 # service template
