diff -urN nagios-plugin-check_raid-4.0.8.org/check_raid_bbu.cfg nagios-plugin-check_raid-4.0.8/check_raid_bbu.cfg
--- nagios-plugin-check_raid-4.0.8.org/check_raid_bbu.cfg	1970-01-01 01:00:00.000000000 +0100
+++ nagios-plugin-check_raid-4.0.8/check_raid_bbu.cfg	2018-12-11 13:49:55.609515941 +0100
@@ -0,0 +1,34 @@
+# Usage:
+# check_raid_bbu
+define command {
+	command_name    check_raid_bbu
+	command_line    /usr/lib/nagios/plugins/check_raid --bbu-monitoring $ARG1$
+}
+
+# service template
+define service {
+	use                     generic-service
+	name                    raid-bbu
+	service_description     raid-bbu
+	register                0
+
+	normal_check_interval   30
+	retry_check_interval    5
+	notification_interval   3600
+
+	check_command           check_raid_bbu
+}
+
+# sample check
+#define service {
+#	use                     raid-bbu
+#	host_name               localhost
+#}
+
+# sample check via nrpe
+#define service {
+#	use                     raid-bbu
+#	hostgroups              linux-servers
+#	host_name               localhost
+#	check_command           check_nrpe!-c check_raid_bbu -t 60
+#}
